internal/store/mutatingwebhookconfiguration.go

```diff
@@ -0,0 +1,110 @@
+/*
+Copyright 2019 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package store
+
+import (
+	admissionregistration "k8s.io/api/admissionregistration/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/watch"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/kube-state-metrics/pkg/metric"
+)
+
+var (
+	descMutatingWebhookConfigurationHelp          = "Kubernetes labels converted to Prometheus labels."
+	descMutatingWebhookConfigurationDefaultLabels = []string{"namespace", "mutatingwebhookconfiguration"}
+
+	mutatingwebhookconfigurationFamilies = []metric.FamilyGenerator{
+		{
+			Name: "kube_mutatingwebhookconfiguration_info",
+			Type: metric.Gauge,
+			Help: "Information about the MutatingWebhookConfiguration.",
+			GenerateFunc: wrapMWCFunc(func(mwc *admissionregistration.MutatingWebhookConfiguration) *metric.Family {
+				return &metric.Family{
+					Metrics: []*metric.Metric{{
+						LabelKeys:   []string{},
```

Comment:
We don't need these. Let's remove it here as well as in `validatewebhookconfiguration` as it's implied. 
```
LabelKeys:   []string{},
LabelValues: []string{},
```

Reply:
thanks